### PR TITLE
[SampleSQLBuilder]Commerce 4993

### PR DESCRIPTION
**Pull request by tinatian**

### CODE REVIEW

**Comment by tinatian** [comment]
ci:test:sf

**Comment by shuyangzhou** [comment]
ci:forward

**Comment by shuyangzhou** [comment]
@vicnate5 please help me to scan the results, thanks!

**Comment by kyle-miho** [comment]
Minium Site is broken in this PR
Steps to Reproduce: Create a New Site using site template "Minium" with this PR (You will see an unexpected error has occured in the UI)
Liferay Log: https://testray.liferay.com/reports/production/logs/2021-04/test-1-5/test-portal-acceptance-pullrequest(master)/7942/functional-tomcat90-mysql57-jdk8/0/3/liferay-log.txt.gz

**Comment by dantewang** [comment]
@dsitu see https://github.com/shuyangzhou/liferay-portal/pull/9943#issuecomment-822882707

File: modules/dxp/apps/commerce-demo-pack/commerce-theme-minium/commerce-theme-minium-full-site-initializer/src/main/resources/com/liferay/commerce/theme/minium/full/site/initializer/internal/dependencies/theme-portlet-settings.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
 		"portletPreferences": {
 			"displayDepth": "0",
 			"displayStyle": {
-				"FileName": "MINIUM_SITE_NAVIGATION.ftl",
+				"FileName": "minium_site_navigation.ftl",
```
Comment (dsitu) [inline modules/dxp/apps/commerce-demo-pack/commerce-theme-minium/commerce-theme-minium-full-site-initializer/src/main/resources/com/liferay/commerce/theme/minium/full/site/initializer/internal/dependencies/theme-portlet-settings.json:20]: "FileName": "MINIUM_SITE_NAVIGATION.ftl",

File: modules/dxp/apps/commerce-demo-pack/commerce-theme-minium/commerce-theme-minium-full-site-initializer/src/main/resources/com/liferay/commerce/theme/minium/full/site/initializer/internal/dependencies/theme-portlet-settings.json [hunk 2]
```diff
@@ -36,7 +36,7 @@
 		"portletPreferences": {
 			"displayDepth": "0",
 			"displayStyle": {
-				"FileName": "MINIUM_SUB_SITE_NAVIGATION.ftl",
+				"FileName": "minium_sub_site_navigation.ftl",
```
Comment (dsitu) [inline modules/dxp/apps/commerce-demo-pack/commerce-theme-minium/commerce-theme-minium-full-site-initializer/src/main/resources/com/liferay/commerce/theme/minium/full/site/initializer/internal/dependencies/theme-portlet-settings.json:39]: Don't think this was meant to be changed.

```suggestion
				"FileName": "MINIUM_SUB_SITE_NAVIGATION.ftl",
```

File: modules/util/portal-tools-sample-sql-builder/src/main/resources/com/liferay/portal/tools/sample/sql/builder/dependencies/data/commerce_theme_portlet_settings.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
 		"portletPreferences": {
 			"displayDepth": "0",
 			"displayStyle": {
-				"FileName": "MINIUM_SITE_NAVIGATION.ftl",
+				"FileName": "minium_site_navigation.ftl",
```
Comment (dsitu) [inline modules/util/portal-tools-sample-sql-builder/src/main/resources/com/liferay/portal/tools/sample/sql/builder/dependencies/data/commerce_theme_portlet_settings.json:8]: Don't think this was meant to be changed.

```suggestion
				"FileName": "MINIUM_SITE_NAVIGATION.ftl",
```

File: modules/util/portal-tools-sample-sql-builder/src/main/resources/com/liferay/portal/tools/sample/sql/builder/dependencies/data/commerce_theme_portlet_settings.json [hunk 4]
```diff
@@ -24,7 +24,7 @@
 		"portletPreferences": {
 			"displayDepth": "0",
 			"displayStyle": {
-				"FileName": "MINIUM_SUB_SITE_NAVIGATION.ftl",
+				"FileName": "minium_sub_site_navigation.ftl",
```
Comment (dsitu) [inline modules/util/portal-tools-sample-sql-builder/src/main/resources/com/liferay/portal/tools/sample/sql/builder/dependencies/data/commerce_theme_portlet_settings.json:27]: Don't think this was meant to be changed.

```suggestion
				"FileName": "MINIUM_SUB_SITE_NAVIGATION.ftl",
```

File: modules/apps/commerce/commerce-theme-minium/commerce-theme-minium-site-initializer/src/main/resources/com/liferay/commerce/theme/minium/site/initializer/internal/dependencies/theme-portlet-settings.json [hunk 5]
```diff
@@ -5,7 +5,7 @@
 		"portletPreferences": {
 			"displayDepth": "0",
 			"displayStyle": {
-				"FileName": "MINIUM_SITE_NAVIGATION.ftl",
+				"FileName": "minium_site_navigation.ftl",
```
Comment (dsitu) [inline modules/apps/commerce/commerce-theme-minium/commerce-theme-minium-site-initializer/src/main/resources/com/liferay/commerce/theme/minium/site/initializer/internal/dependencies/theme-portlet-settings.json:8]: Don't think this was meant to be changed.

```suggestion
				"FileName": "MINIUM_SITE_NAVIGATION.ftl",
```

File: modules/apps/commerce/commerce-theme-minium/commerce-theme-minium-site-initializer/src/main/resources/com/liferay/commerce/theme/minium/site/initializer/internal/dependencies/theme-portlet-settings.json [hunk 6]
```diff
@@ -24,7 +24,7 @@
 		"portletPreferences": {
 			"displayDepth": "0",
 			"displayStyle": {
-				"FileName": "MINIUM_SUB_SITE_NAVIGATION.ftl",
+				"FileName": "minium_sub_site_navigation.ftl",
```
Comment (dsitu) [inline modules/apps/commerce/commerce-theme-minium/commerce-theme-minium-site-initializer/src/main/resources/com/liferay/commerce/theme/minium/site/initializer/internal/dependencies/theme-portlet-settings.json:27]: Don't think this was meant to be changed.

```suggestion
				"FileName": "MINIUM_SUB_SITE_NAVIGATION.ftl",
```

**Comment by dantewang** [comment]
@shuyangzhou @tinatian As @dsitu suggested I dropped the last commit (SF, rename) and sent to https://github.com/shuyangzhou/liferay-portal/pull/9967

**Comment by tinatian** [comment]
Fix and resend at https://github.com/shuyangzhou/liferay-portal/pull/9968.

**Pull request closed without merging**